src/__tests__/uninstallAddon.spec.js

```diff
@@ -0,0 +1,42 @@
+import { mockAddonObj, mockConfig, mockInstallAddonsDict } from './mock'
+import { uninstallAddon } from '../uninstallAddon'
+import fs from 'fs'
+
+describe('uninstallAddon function', () => {
+    describe('GIVEN a valid configObj', () => {
+        describe('GIVEN a valid installAddonsDict', () => {
```

Comment:
`And a valid installAddonsDict`

Reply:
more tests to be added in when integrityCheck.js is refactored